spec/controllers/v1/sessions_controller_spec.rb

```diff
@@ -547,7 +554,7 @@ def expect_logger_msg(level, msg)
 
         it 'redirects to identity proof URL', :aggregate_failures do
           Timecop.freeze(Time.current)
-          expect_any_instance_of(SAML::PostURLService).to receive(:should_uplevel?).twice.and_return(true)
+          expect_any_instance_of(SAML::PostURLService).to receive(:should_uplevel?).once.and_return(true)
```

Comment:
@bramleyjl I wouldn't have expected this test case to change

Reply:
In `saml_cookie_content` we removed the more complicated transaction_id check that included a `should_uplevel` call in favor of the Redis cache, so now `should_uplevel` is only being called once, inside `user_login`

Reply:
ah thats right, thanks for pointing that out